.github/CODEOWNERS

```diff
@@ -0,0 +1,3 @@
+@richardcase
+@alexander-demicev
+@belgaied2
```

Comment:
should we add a new line and all other files?

Reply:
Goo dpoint, added new lines.